### PR TITLE
[Vengeance] initial pass at passive talents

### PR DESCRIPTION
**Pull request by ToppleTheNun**

### CODE REVIEW

File: engine/class_modules/sc_demon_hunter.cpp [hunk 1]
```diff
@@ -1345,6 +1345,21 @@ class demon_hunter_action_t : public Base
...
+    ab::apply_affecting_aura( p->talent.vengeance.roaring_fire );
+    ab::apply_affecting_aura( p->talent.vengeance.agonizing_flames );
+    ab::apply_affecting_aura( p->talent.vengeance.extended_spikes );
+    ab::apply_affecting_aura( p->talent.vengeance.burning_blood );
+    ab::apply_affecting_aura( p->talent.vengeance.ruinous_bulwark );
+    ab::apply_affecting_aura( p->talent.vengeance.volatile_flameblood );
+    ab::apply_affecting_aura( p->talent.vengeance.chains_of_anger );
+    ab::apply_affecting_aura( p->talent.vengeance.stoke_the_flames );
+    // Only handles the maximum stacks for Frailty
+    ab::apply_affecting_aura( p->talent.vengeance.soulcrush );
```
Comment (EvanMichaels) [inline engine/class_modules/sc_demon_hunter.cpp]: Max stacks have to be applied to the buff. debuffs.frailty->apply_affecting_aura

Comment (ToppleTheNun) [reply engine/class_modules/sc_demon_hunter.cpp]: I'll handle that in a separate PR then and address more frailty things at the same time.

**Comment by ToppleTheNun** [comment]
Once this is good to go, I have a Void Reaver/Frailty/Soulcrush change to follow up.

**Comment by ToppleTheNun** [comment]
Closing this one since the other one is being reviewed.

**Pull request closed without merging**